### PR TITLE
Updated Schema Profuctor readme with another example

### PR DESCRIPTION
**Pull request by elland**
## Checklist

 - [x] The **PR Title** explains the impact of the change.
 - [x] The **PR description** provides context as to why the change should occur and what the code contributes to that effect. This could also be a link to a JIRA ticket or a Github issue, if there is one.

### CODE REVIEW

**Review by fisx — CHANGES_REQUESTED** [review]
please change as discussed in https://github.com/wireapp/wire-server/pull/2564

**Review by pcapriotti — COMMENTED** [review]
I'm not sure this example requires `mkSchema`. Couldn't you use [`tag`](https://wearezeta.atlassian.net/wiki/spaces/CORE/pages/553418860/MLS+implementation+in+wire-server#Backend-side-removal) and `withParser`?

File: libs/schema-profunctor/README.md [hunk 1]
```diff
@@ -203,6 +203,50 @@ Finally, we add a name to the schema using the `named`
...
+              else A.parseFail "Expected ''unlimited' or '0'."
+
+      parseSeconds :: A.Value -> A.Parser FeatureTTL
+      parseSeconds = A.withScientific "FeatureTTL" $
+        \s -> case toBoundedInteger s of
+          Just 0 -> pure FeatureTTLUnlimited
+          Just i -> pure . FeatureTTLSeconds $ i
+          Nothing -> A.parseFail "Expected an integer."
+
+      fromTTL :: FeatureTTL -> Maybe A.Value
```
Comment (fisx) [inline libs/schema-profunctor/README.md:244]: Btw @pcapriotti why is this a `Maybe`?

Comment (pcapriotti) [reply libs/schema-profunctor/README.md:244]: Good question. In order to support modular serialisation of sum types, serialisers can "fail" by returning `Nothing`. So for example, consider the serialiser side of how we build a schema for `A + B` from schemas `sa`, `sb` for `A` and `B` respectively. We first turn the serialiser of `sa` into a serialiser for `A + B` that simply fails in the `B` case, and similarly for `sb`. Then finally we merge them together using the appropriate `Monoid` instance for functions into `Maybe`. [This](https://github.com/wireapp/wire-server/blob/develop/libs/schema-profunctor/README.md#sum-types) has a bit more detail on the user-facing API of this mechanism.

File: libs/schema-profunctor/README.md [hunk 2]
```diff
@@ -203,6 +203,50 @@ Finally, we add a name to the schema using the `named`
...
+        A.withText "FeatureTTL" $
+          \t ->
+            if t == "unlimited" || t == "0"
+              then pure FeatureTTLUnlimited
+              else A.parseFail "Expected ''unlimited' or '0'."
+
+      parseSeconds :: A.Value -> A.Parser FeatureTTL
+      parseSeconds = A.withScientific "FeatureTTL" $
+        \s -> case toBoundedInteger s of
+          Just 0 -> pure FeatureTTLUnlimited
```
Comment (fisx) [inline libs/schema-profunctor/README.md:240]:
```suggestion
          Just 0 -> pure FeatureTTLUnlimited  -- this makes `decode . encode /= id`, but you get the idea.
```

**Review by fisx — APPROVED** [review]

**Pull request merged**